sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectView.groovy

```diff
@@ -354,4 +309,69 @@ class ProjectView extends ProjectDesign {
         DataProvider<ProjectSummary, ?> dataProvider = this.projectGrid.getDataProvider()
         return dataProvider
     }
+
+    /**
+     * Adds responsiveness to an abstractComponent
+     *
+     * <p>This applies the css class style .responsive-grid-layout to the provided abstractComponent allowing it to display it's content in a responsive manner</p>
+     *
+     * @param AbstractComponent the {@link com.vaadin.ui.AbstractComponent}, where the css style and responsiveness should be added
+     * @since 1.0.2
+     */
+    static void setupLayoutResponsiveness(AbstractComponent abstractComponent) {
+        abstractComponent.addStyleName("responsive-grid-layout")
+        abstractComponent.setWidthFull()
+    }
+
+    /**
+     * Disables manual resizing of individual grid columns and calculates column width dependent on screen size
+     *
+     * @param isDynamicResizing boolean value determining if a grid should be manually or automatically resizable
+     * @since 1.0.2
+     */
+    private void setDynamicResizing(boolean isDynamicResizing) {
```

Comment:
no passing of internal states necessary in internal methods that know the state

Reply:
Sorry, I also dont get the purpose of the method tbh. Just set a default init when the constructor is called